From 7b613535cda35020e6f65a96372fc0e9200f8d64 Mon Sep 17 00:00:00 2001
From: Fabian Pflug <f.pflug@pengutronix.de>
Date: Tue, 21 Oct 2025 12:28:32 +0200
Subject: [PATCH 1/2] common: deep-probe: include of for of_device_id

The struct of_device_id is used by deep-probe in its own struct
deep_probe_entry, but never included. It does work, because all files
needing deep-probe currently also include of.h, but that should not be
mandatory.

Signed-off-by: Fabian Pflug <f.pflug@pengutronix.de>
---
 include/deep-probe.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/deep-probe.h b/include/deep-probe.h
index 9ba9d09ec3..d1413f979e 100644
--- a/include/deep-probe.h
+++ b/include/deep-probe.h
@@ -5,6 +5,7 @@
 #include <linux/stringify.h>
 #include <linux/compiler_types.h>
 #include <linux/types.h>
+#include <of.h>
 
 struct deep_probe_entry {
 	const struct of_device_id *device_id;
-- 
2.47.3

